.github/workflows/unittest.yml

```diff
@@ -0,0 +1,27 @@
+name: unit-test
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [next, qa, main]
+
+jobs:
+  build-wheel:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        auto-update-conda: true
+        mamba-version: "*"
+        environment-file: environment.yml
+    - name: Install quicknxs via setup.py
```

Comment:
I think the original instructions were a bit off. This job should actually be
```sh
python setup.py bdist_wheel
check-wheel-contents dist/*.whl
```
the python `build` module apparently doesn't know how to use `setup.py`

Reply:
updated with the suggested edits